src/fiskaltrust.ifPOS/v1/de/Models/TseInfo.cs

```diff
@@ -6,28 +6,39 @@ namespace fiskaltrust.ifPOS.v1.de
     [DataContract]
     public class TseInfo
     {
-        [DataMember(Order = 1)]
+        [DataMember(Order = 10)]
         public long MaxNumberOfClients { get; set; }
 
-        [DataMember(Order = 2)]
+        [DataMember(Order = 20)]
         public long CurrentNumberOfClients { get; set; }
 
-        [DataMember(Order = 3)]
+        [DataMember(Order = 30)]
+        public IEnumerable<string> CurrentClientIds { get; set; }
+
+        [DataMember(Order = 40)]
         public long MaxNumberOfTransactions { get; set; }
 
-        [DataMember(Order = 4)]
+        [DataMember(Order = 50)]
         public long CurrentNumberOfTransactions { get; set; }
 
-        [DataMember(Order = 5)]
+        /*
+        [DataMember(Order = 60)]
         public IEnumerable<CertificateInfo> Certificates { get; set; }
 
-        [DataMember(Order = 6)]
+        [DataMember(Order = 70)]
         public IEnumerable<string> SerialNumbersBase64 { get; set; }
+        */
+
+        [DataMember(Order = 60)]
+        public string TsePublicKeyBase64 { get; set; }
+
+        [DataMember(Order = 70)]
+        public string TseSerialNumberOctet { get; set; }
```

Comment:
If we have multiple certificates we need to (maybe additionally?) add a list with serialnumbers

Reply:
see https://github.com/fiskaltrust/middleware-interface-dotnet/pull/4#discussion_r417946263